packages/realm_dart/test/sync_migration_test.dart

```diff
@@ -0,0 +1,259 @@
+// Copyright 2022 MongoDB, Inc.
```

Comment:
Mostly a question. Shouldn't this say 2024? Ie. the year the file was first added.

Reply:
It should 👀